lib/modules/manager/npm/extract/yarn.ts

```diff
@@ -121,3 +125,55 @@ export function getYarnVersionFromLock(lockfile: LockFile): string {
 
   return '^2.0.0';
 }
+
+export async function extractYarnCatalogs(
+  catalogs: YarnCatalogs | undefined,
+  packageFile: string,
+  hasPackageManager: boolean,
+): Promise<PackageFileContent<NpmManagerData> | null> {
+  logger.trace(`yarn.extractYarnCatalogs(${packageFile})`);
+
+  const yarnCatalogs = yarnCatalogsToArray(catalogs);
+
+  const deps = extractCatalogDeps(yarnCatalogs, 'yarn');
+
+  let yarnLock: string | undefined;
+  const filePath = getSiblingFileName(packageFile, 'yarn.lock');
+
+  if (await readLocalFile(filePath, 'utf8')) {
```

Comment:
```suggestion
  if (await localPathExists(filePath, 'utf8')) {
```

isn't this enough? we don't need the contents and a later update will fail if it is somehow broken